### PR TITLE
Make it clear that `persistStore` should be awaited

### LINKED ISSUE #62: [DOCS] Docs should state that persistCache should be awaited
Hey,

It is not clear in the documentation that `persistCache` is async and should be awaited.

```js
await persistCache({
  storage: window.localStorage,
  cache,
});
```

Removing `await` breaks persistence, because the first queries will resolve before the cache is persisted, and won't resolve again when it is.

Could you please confirm that this is by design (or maybe something else is causing this and I'm wrong), and tell me if I should open a PR to clarify this in the docs?

Thanks!

### PR DESCRIPTION
**Pull request by adrienharnay**
Fixes #62

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +50,7 @@ import { persistCache } from 'apollo-cache-persist';
 
 const cache = new InMemoryCache({...});
 
+// await before instantiating ApolloClient, else queries might run before the cache is persisted
 persistCache({
```
Comment (wtrocki) [inline README.md]: I would simply add await to code without comment.

File: README.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ import { persistCache } from 'apollo-cache-persist';
 
 const cache = new InMemoryCache({...});
 
+// await before instantiating ApolloClient, else queries might run before the cache is persisted
```
Comment (wtrocki) [inline README.md]: Same

**Comment by adrienharnay** [comment]
Thank you for the edit 👍

**Comment by wtrocki** [comment]
💯 Awesome contribution. Thank you for making Apollo Community a great place!

**Pull request merged**